sinker/service.go

```diff
@@ -221,6 +226,19 @@ func (svc sinkerService) handleMetrics(agentID string, channelID string, subtopi
 			if err != nil {
 				svc.logger.Warn(fmt.Sprintf("unable to remote write to sinkID: %s", id), zap.String("policy_id", m.PolicyID), zap.String("agent_id", agentID), zap.String("owner_id", agent.OwnerID), zap.Error(err))
 			}
+
+			// send operational metrics
+			labels := []string{
+				"method", "sinker_payload_size",
+				"agent_id", agentID,
+				"agent", agent.AgentName,
+				"policy_id", m.PolicyID,
+				"policy", m.PolicyName,
+				"sink_id", id,
+				"owner_id", agent.OwnerID,
+			}
+			svc.counter.With(labels...).Add(1)
+			svc.gauge.With(labels...).Add(float64(unsafe.Sizeof(m)))
```

Comment:
I don't think we need "unsafe" here - the string size of the body should be fine (which we debug print in other places)